sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusClientBuilderTest.java

```diff
@@ -205,6 +243,42 @@ public void testProxyOptionsConfiguration(String proxyConfiguration, boolean exp
 
     }
 
+    /**
+     * Connection is shared between senders and receivers when created with same builder.
+     */
+    @Test
+    void sharedConnectionTest() {
```

Comment:
I'm trying to understand what this exercises other than you can build a client? Our normal integration tests pass-through this scenario and verify we create one connection.

I was hoping to exercise a shared connection by actually creating an integration test with consumers and producers.

Reply:
removed